Kong/docs.konghq.com
# Release: Mesh 2.0

Release roll-up PR for Mesh 2.0 release.

DO NOT COMMIT DIRECTLY TO THIS BRANCH. THIS PR IS FOR QA AND PREVIEW.

Please open PRs against `release/mesh-2.0`.

Preview: https://deploy-preview-4693--kongdocs.netlify.app/mesh/latest/

### Release checklist
- [x] Release branch 
- [x] Changelog (should've been updated by the dev team, double check against Kuma changelog) @cloudjumpercat 
- [x] Create release PR (Diana)
- [x] Update nav so shared content is pointing to the 2.0 version folder instead of the /dev folder (this can only be done once the Kuma team does the folder conversion) @cloudjumpercat #4707 

### Testing/QA
Do the following in your section: 

* Visual check that content renders correctly (includes images, code blocks, etc.)
* Click through the entire Mesh nav and ensure all pages exist in Kuma (verify against the 2.0.x docs cut: https://deploy-preview-1124--kuma.netlify.app/)
* Check links resolve correctly. 
* Check CNCF says "Kuma" and not "mesh" (if applicable)

- [x] Introduction (Lena)
- [x] Getting Started (Lena)
- [x] Install (Lena)
- [x] Deploy  (angel)
- [x] Explore (angel)
- [x] Networking @cloudjumpercat 
- [x] Security (angel) 
- [x] Monitor and Manage @cloudjumpercat 
- [x] Policies @cloudjumpercat 
- [x] Enterprise Features (Lena)
- [x] Reference Docs @cloudjumpercat (special note for this one, in the Commands subsection, those pages link to unlisted, generated pages, so make sure to check that those render correctly)

Other misc tasks:
- [x]  remove line 17 in this [file](https://github.com/Kong/docs.konghq.com/blob/release/mesh-2.0/app/_plugins/generators/kuma_to_kong_mesh.rb#L17) (Lena).
- [x] Redirect the /community link since it only applies to Kuma (Angel)
- [x] make a policies page (Lena)
